src/Ninja/G1CP/Content/Misc/info.d

```diff
@@ -84,3 +84,12 @@ funt int G1CP_ChangeInfoDesc(var string infoName, var string originalStr, var st
 
     return (G1CP_ReplaceAssignStr(infoId, 0, "C_Info.description", 0, originalStr, correctedStr) > 0);
 }
+
+/*
+ * Change the value of an info's integer variable
+ */
+funt int G1CP_ChangeInfoIntVar(var string infoName, var string infoProp, var int originalInt, var int correctedInt) {
```

Comment:
Typo here: func instead of funt

Right now, the function contains almost the identical set of parameters of the function that it substitutes. I am not so sure about the gains here - considering it only affects two fixes so far, that are already very concise.

Depending whether there will be more such fixes, we could decide to make two functions instead to reduce the number of parameters and increase their relevance:
```D
func int G1CP_ChangeInfoPermanent(var string infoName, var int originalOn, var int correctedOn)

func int G1CP_ChangeInfoTrade(var string infoName, var int originalOn, var int correctedOn)
```
However, this I would only propose if there is an substantial amount of such fixes.

Reply:
Thanks for the typo hint!

I understand the motivation behind splitting the functions, but like you I don't think they are necessary at the moment. Definitely worth to revisit in the future!